Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright 2015-2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates a new empty dataset in the specified project.
+    /// </para>
+    /// <para type="description">
+    /// Creates a new empty dataset in the specified project.  This command can either take a premade Dataset object, 
+    /// or build one using parameters.  Premade Datasets can be passed via the pipeline or with the -Dataset parameter.
+    /// If both are supplied, the premade dataset object will be used and the builder values will be ignored.  If no 
+    /// Project is specified, the default project will be used.  This cmdlet returns a Dataset object.  
+    /// The builder option will create a new Dataset object and inject all of the supplied values before insertion. 
+    /// Required build parameters are passed directly after the command, whereas optional parameters are passed as flags.
+    /// -- Builder parameters: 
+    /// DatasetId (required) - unique identifier for the dataset.
+    /// Name - descriptive name for the dataset.
+    /// Description - user-friendly description of the dataset.
+    /// Timeout - default duration in ms for tables in the dataset to exist.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> $dataset | New-GcbqDataset -Project $projId </code>
+    ///   <para>This command will take a premade Dataset object from the pipeline and insert it into the Cloud project. 
```

Comment:
Instead of "Cloud project. project $projectId", should be "Cloud project "$projectId"". Also, I think instead of $projId, you should use "my-project" or something like that.